SafeguardCommon/Src/Modules/CommonLogger.pqm

```diff
@@ -198,16 +198,31 @@ let
         in
             if isTesting = null or isTesting = false then
                 Diagnostics.Trace(
-                    logLevel, prefix & ": " & (try _valueToText(value) otherwise "<error getting value>"), value,
+                    logLevel, "[SafeguardPowerBI]" & prefix & ": " & (try _valueToText(value) otherwise "<error getting value>"), value,

```

Comment:
TestCommonLogger.query.pq needs to be updated.

SafeguardCommon can be built with make build (binaries needs to be installed on the system as it does not have a bootstrap yet) and tests can be run with make unit-tests.

Reply:
Resolved by 

https://github.com/OneIdentity/SafeguardPowerBI/pull/68/commits/3eaa8e5f43b3de962ddfe30a6606694e7d6dfbe8